ios/Sources/Component/Card/SmallCard.swift

```diff
@@ -34,127 +36,96 @@ public struct SmallCard: View {
     }
 
     public var body: some View {
-        GeometryReader { geometry in
-            VStack(alignment: .leading, spacing: 16) {
-                ImageView(
-                    imageURL: imageURL,
-                    placeholder: .noImage,
-                    placeholderSize: .small,
-                    width: geometry.size.width,
-                    height: geometry.size.width * 114/163
-                )
+        VStack(alignment: .leading, spacing: 16) {
+            ImageView(
+                imageURL: imageURL,
+                placeholder: .noImage,
+                placeholderSize: .small,
+                width: Const.imageViewWidth,
+                height: Const.imageViewWidth * 114/163
+            )
 
-                VStack(alignment: .leading, spacing: 12) {
-                    VStack(alignment: .leading, spacing: .zero) {
-                        Text(title)
-                            .font(.subheadline)
-                            .foregroundColor(AssetColor.Base.primary.color)
-                            .lineLimit(3)
-                            .frame(maxHeight: .infinity, alignment: .top)
+            VStack(alignment: .leading, spacing: 12) {
+                Group {
+                    Text(title)
+                        .font(.subheadline)
+                        .foregroundColor(AssetColor.Base.primary.color)
+                        .lineLimit(3)
+                        .frame(maxHeight: .infinity, alignment: .top)
 
-                        Spacer(minLength: 4)
+                    Spacer(minLength: 4)
 
-                        Text(date.formatted)
-                            .font(.caption)
-                            .foregroundColor(AssetColor.Base.tertiary.color)
-                    }
+                    Text(date.formatted)
+                        .font(.caption)
+                        .foregroundColor(AssetColor.Base.tertiary.color)
+                }
```

Comment:
Why is this wrapped with Group?

Reply:
Current code was `VStack` with spacing zero. If there's any spacing `Group` is minimum view builder.
Also if I remove Group from this code, spacing 12 will added to each texts.